### ISSUE TITLE
Insulin, SMB, treatments and Carbs not received from AAPS

### ISSUE THREAD

**Issue by amosxo** [NightscoutFoundation/xDrip#1716]
The SMB symbols and treatments are no longer transmitted from AAPS to xdrip+. This happened in the middle o the day of February the 20th 2021. It was working normally before for more than 2 months. The treatments are visible thought in AAPS. I had not changed the settings in either of the apps the day it  stopped transmitting treatments and insulin from AAPS to xdrip+. In xdrip+ interapp settings the local broadcast and accept treatments are enabled. 

I am using AAPS version 2.8.2 full release and 17cbcf0-2021.03.08 xdrip+ on Redmi note pro miui 12.0.2.0 android 10 QKQ1.191215.002. 

I tried many options and I have rebuild and reinstalled AAPS and xdrip but it did not help.
Unfortunately the log before the issue occurred is now gone but I can provide the current log.

Also in nigtscout I can generate reports with treatments and insulin until the day it happened. After this day no insulin or carb are reported.

**Comment by reijoahola-nightscout** [comment]
I have noticed that AAPS -> xDrip+ broadcast do not work when AAPS -> NS connection is not working. Do you have this problem in NS connection ?

**Comment by amosxo** [comment]
You are right. That was it! For some reason the NS client in AAPS was paused. Maybe I did this after a connection problem causing the issue you mentioned and I was thinking that there was an interference from the NS client to the interapp connection causing this. I have unpaused and restated and after doing this several times it finally started working again: Now I see the treatments in both NS and xdrip. This is definitely the issue you found. The interapp connection should be independent from the NS client connection. Let's hope tha this issue will be one day fixed. Since there is already an other ticket about this I will go on and close my ticket. Thank you.
PS: I have sometimes observed that I do not receive BG data from G6 sensor in xdrip when I do not have internet connection. This might be related to this issue but it is not so annoying since I almost always have internet access. I also need to record my data in NS client for generating reports therefor I never turn off the data transfer on my phone.
